src/components/nav/MobileSideBar.js

```diff
@@ -0,0 +1,113 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+import MuiAppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import CssBaseline from "@mui/material/CssBaseline";
+import Divider from "@mui/material/Divider";
+import Drawer from "@mui/material/Drawer";
+import IconButton from "@mui/material/IconButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import { styled, useTheme } from "@mui/material/styles";
+
+const drawerWidth = 190;
+
+const AppBar = styled(MuiAppBar)(({ theme }) => ({
+  backgroundColor: theme.palette.primary.main,
+  border: `2px solid ${theme.palette.secondary.main}`,
+  borderRadius: `0 0 12px 12px`,
+  maxWidth: theme.breakpoints.lg,
+  //display: "flex",
+  //alignItems: "center",
+}));
+
+const MobileSideBar = ({ title, drawer }) => {
+  const [mobileOpen, setMobileOpen] = React.useState(false);
+  const theme = useTheme();
+
+  const handleDrawerToggle = () => {
+    setMobileOpen(!mobileOpen);
+  };
+
+  return (
+    <Box>
+      <CssBaseline />
+      <AppBar
+        position="fixed"
+        sx={{
+          width: { sm: `calc(100% - ${drawerWidth}px)` },
+          ml: { sm: `${drawerWidth}px` },
+        }}
+      >
+        <Toolbar sx={{ display: "flex", justifyContent: "space-between" }}>
+          <IconButton
+            color="inherit"
+            aria-label="open drawer"
+            edge="start"
+            onClick={handleDrawerToggle}
+            sx={{ mr: 2, display: { sm: "none" } }}
+          >
+            <MenuIcon />
```

Comment:
It might be an idea to toggle this with a `import CloseIcon from '@mui/icons-material/Close';` from the icons library based on the state of `mobileOpen`